### ISSUE TITLE
JetEngine + Grid Gallery => Lack of possibility to set ratio for images

### ISSUE THREAD

**Issue by ConsultMePL** [CrocoBlock/suggestions#3596]
Hi,

With JetGridGallery follows pretty much a default's Elementor gallery, apart from one crucial function: ability to set different ratio for images.

If I create w CPT with galleries and want to display them on the front end, the only way to keep same ratio for images is to you Post Thumbnail 300x300. The issue with this one is that its 300x300 so very low quality. When you have a gallery with horizontal/square images and vertical ones, the gallery is a mess.

It doesn't seem to be complicated to create such function, which is available in Elementor's default gallery widget. It's not even that complicated to change it using CSS, just need to find a right class. 

It would be great if you guys would implement this function.

Thanks,
Dominik

**Comment by GDUpson** [comment]
Yes please this is a must! The gallery slider is useless without the ability justify the height of the images.
